fix: ProductsPage useProductsQuery 타입 오류 수정

## 🐞 버그 설명

<!-- 어떤 문제가 발생했는지 명확하게 써주세요. (예: 로그인 버튼 클릭 시 아무 반응 없음) -->
`src/pages/ProductsPage.tsx(10,67): error TS2559: Type 'string' has no properties in common with type 'UseProductsParams'`라는 빌드 오류가 뜸

## 🧠 원인 추정 (선택)

<!-- 대충 어디서 문제가 생겼을 것 같은지 (예: 상태 관리, 비동기 처리 등) -->

## 🔗 관련 이슈/PR

<!-- 관련된 이슈나 참고 링크 -->

--- a/src/pages/ProductsPage.tsx
+++ b/src/pages/ProductsPage.tsx
@@ -7,7 +7,7 @@ import { useMemo, useState } from 'react';
 export function ProductsPage() {
   const { searchTerm } = useSearchStore();
   const [sortOption, setSortOption] = useState('');
-  const { data: products, isLoading, isError } = useProductsQuery(sortOption);
+  const { data: products, isLoading, isError } = useProductsQuery({ sortOption });
 
   const filteredProducts = useMemo(() => {
     if (!products) return [];

